### PR TITLE
Added an empty dialog for Locating points in shape file

### PR DESCRIPTION
**Pull request by HawardKetoyoMsatsi**
@maxwellfundi @shadrackkibet this is ready for review.
It contains frmMain changes which should be merged prior to opening another PR.

### CODE REVIEW

File: instat/frmMain.vb [hunk 1]
```diff
@@ -2346,6 +2342,13 @@ Public Class frmMain
...
         dlgOtherRosePlots.ShowDialog()
+    End Sub
+
+    Private Sub mnuStationMaps_Click(sender As Object, e As EventArgs) Handles mnuStationMaps.Click
+        dlgClimaticStationMaps.ShowDialog()
+    End Sub
 
+    Private Sub mnuCheckStations_Click(sender As Object, e As EventArgs) Handles mnuCheckStations.Click
+        dlgLocatingPointsInShapeFile.ShowDialog()
     End Sub
```
Comment (shadrackkibet) [inline instat/frmMain.vb:2353]: Change the naming convention. You should start with the menu name then sub-menu, dialog etc. check other places for examples

**Comment by dannyparsons** [comment]
@HawardKetoyoMsatsi Can you make the menu items invisible until the dialog is implemented?

File: instat/frmMain.vb (hunk 1 above)
Comment (HawardKetoyoMsatsi) [reply instat/frmMain.vb]: @shadrackkibet the above has been fixed

**Review by shadrackkibet — APPROVED** [review]

**Review by maxwellfundi — APPROVED** [review]

**Comment by shadrackkibet** [comment]
@dannyparsons please merge if you are happy with this, then we can proceed with the task.

**Comment by lloyddewit** [comment]
I reviewed and I thought it was OK, so I went ahead and merged to avoid delay. I hope that's OK

**Comment by shadrackkibet** [comment]
Thanks @lloyddewit

**Pull request merged**